### PR TITLE
fix(card): expand body instead of header and footer

### PR DESCRIPTION
**Pull request by prima-lseg**
## Description
Change card styling in body part to expand size instead of expanding header and footer.

Fixes # (issue)
[DME-18155](https://jira.refinitiv.com/browse/DME-18155)
## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/card/index.ts [hunk 1]
```diff
@@ -49,13 +49,26 @@ export class Card extends BasicElement {
...
         flex: 1;
         min-width: 0px;
       }
+
+      [part~='body'] {
+        overflow: auto;
+        flex: 1;
+      }
+
+      [part~='footer'] {
```
Comment (bualoy-napat) [inline packages/elements/src/card/index.ts:68]:
```suggestion
      [part~='header'],
      [part~='footer'] {
```

File: packages/elements/src/card/index.ts [hunk 2]
```diff
@@ -49,13 +49,26 @@ export class Card extends BasicElement {
         display: flex;
         flex-flow: column nowrap;
       }
+
       [part~='header'] {
         display: flex;
+        flex: 0 0 auto;
```
Comment (bualoy-napat) [inline packages/elements/src/card/index.ts]:
```suggestion
```

**Comment by bualoy-napat** [comment]
LGTM

**Comment by Theeraphat-Sorasetsakul** [comment]
Verified.

**Review by bualoy-napat — APPROVED** [review]

**Pull request merged**